shared/js/dataDriver.js

```diff
@@ -417,6 +426,13 @@ loop.DataDriver = function() {
             userId: id
           };
           break;
+
+        case "tocPage":
+          dispatchAction = "ReceivedTile";
+          dispatchExtra = {
+            tile: data.tile
```

Comment:
There is no `data.tile` so this probably isn't doing what you're expecting it to do.